### ISSUE TITLE
Ensure visible label and accessible name match for hamburger menu

### ISSUE THREAD

**Issue by keelanfh** [localgovdrupal/localgov_base#377]
### Impact

Users of voice recognition software will speak the visible label in order to interact with elements – for example ‘Click Menu’. However, if the underlying accessible name does not match the command will not be recognised 

### WCAG Fail 

This item is a Fail of the WCAG 2.1 AA guidelines

### Expected results

User of voice recognition software speaks ‘Click menu’ and the hamburger menu is activated and menu content expands

### Suggested fix

Remove the aria-label ('toggle primary navigation') from the <button> element 

### Steps to reproduce

Load page, reduce viewport until mobile menu is shown, open developer tools and inspect the element. Notice the aria-label

![image](https://user-images.githubusercontent.com/19519457/219048205-b0ff0b3d-3991-4adb-bf4b-5a5cb4196802.png)

**Comment by markconroy** [comment]
PR Ready
